### ISSUE TITLE
Editorial issues from Ben

### ISSUE THREAD

**Issue by cabo** [core-wg/coap-tcp-tls#158]
```
10.2: Is the registration policy "analogous to" that of [RFC7252] S12.2,
or
"identical to" it. If the answer is not "identical", then the policy
should be detailed here.
```

It is analogous, because the structure of the subregistry is slightly different (additional column).
We can add a sentence that the value of the additional column does not influence the choice of the policy.

```
Editorial:

5.3: "One CSM MUST be sent by both endpoints...": s/both/each
```

Good point.

```
7.6: The "updates" in this section are confusing. I understand this to
mean
that the procedures for TCP and WS are identical to those for UDP except
for
the mentioned steps. But the language of the form of "This step from
[RFC7252] is updated to:" makes it sound like this intends to actually
change
the language in 7252 to this new language. If the latter, then that
effectively removes UDP support from 7252 as updated.
```

OK, “update” is not the right word them.

```
This could easily be fixed by changing that to something to the effect
of
"When using TCP, this step changes to …"
```

Good point.

```
Appendix A: Why is this an appendix? Updates to a standards track RFC
seem to
warrant a more prominent position in the draft.
```

It was smeared all over the document, and then we collected it in an appendix.
I don’t think anyone would have a strong opposition against making it a mainline section.
Should we?

**Comment by brianraymor** [comment]
In general, I prefer to separate out unrelated issues (editorial or not) to simplify tracking/closing:

Regarding:

`5.3: "One CSM MUST be sent by both endpoints...": s/both/each`

There are at least three similar cases of "both endpoints" which would need to be corrected together.

**Comment by brianraymor** [comment]
@cabo - Regarding Ben's comment (which you seem to concur with ...):

_7.6: The "updates" in this section are confusing. I understand this to mean that the procedures for TCP and WS are identical to those for UDP except for the mentioned steps. But the language of the form of "This step from [RFC7252] is updated to:" makes it sound like this intends to actually change the language in 7252 to this new language. If the latter, then that effectively removes UDP support from 7252 as updated._

_This could easily be fixed by changing that to something to the effect of "When using TCP, this step changes to ..."_

The text actually updates RFC7252 to address all cases. I don't see how it _**effectively removes UDP support**_ since the text is additive. For example:

 This step from [RFC7252]:
   7.  If |port| does not equal the request’s **destination UDP port**, include a Uri-Port Option and let that 
        option’s value be |port|.
   is updated to:
   7.  If |port| does not equal the request’s **destination UDP port or TCP port,** include a Uri-Port Option 
        and let that option’s value be |port|.

**Comment by brianraymor** [comment]
@cabo - I do not feel strongly about moving Appendix A - but I'm open to suggestions. It's hard to find the right place in the flow - perhaps before/after Section 6?

**Comment by brianraymor** [comment]
This issue is partially addressed by #180 - which I'm merging now since I'm on leave for the next two weeks. The other potential editorial changes are pending responses from @cabo.
